data/io.elementary.files.appdata.xml.in.in

```diff
@@ -58,6 +58,25 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="Next" date="Future" urgency="medium">
+      <description>
+        <p>New features:</p>
+        <ul>
+          <li>Initial git plugin support</li>
+          <li>Follow global history setting when restoring and saving tabs</li>
+        </ul>
+        <p>Other updates:</p>
+        <ul>
+          <li>Ensure tabs showing same folder share an Async object</li>
```

Comment:
I'm not sure I understand what this means. Can you clarify?

Reply:
How about "Ensure tabs showing same folder remain synchronized". This was a fix for #1095. I guess you do not want to reference issue numbers in appdata?

Reply:
Yeah I think that description is much clearer. And yes issue numbers probably don't mean anything to most people